app/assets/javascripts/student_profile/record_service.jsx

```diff
@@ -64,22 +66,43 @@ import serviceColor from '../student_profile/service_color.js';
 
     getInitialState: function() {
       const {nowMoment} = this.props;
+      
       return {
         serviceTypeId: null,
-        providedByEducatorName: "",
+        providedByEducatorName: ''  ,
         dateStartedText: nowMoment.format('MM/DD/YYYY'),
-        estimatedEndDateText: nowMoment.format('MM/DD/YYYY')
+        estimatedEndDateText: this.defaultEstimatedEndDate(nowMoment).format('MM/DD/YYYY')
       };
     },
 
-    dateStartedMoment: function() {
-      const {dateStartedText} = this.state;
-      return moment.utc(dateStartedText, 'MM/DD/YYYY', true); // strict parsing
+    // The default is to assume the service will last until the end of the school year.
```

Comment:
This isn't exact, and may differ by district or by particular programs (eg that are year-round).